scripts/docker_clone_volume.sh

```diff
@@ -0,0 +1,75 @@
+#!/bin/bash
```

Comment:
is this file needed/used?

Reply:
not needed now but when we were getting going I used it a couple of times to duplicate data volumes.  But yeah, it's fairly googleable

Reply:
cool, no harm in keeping it around, maybe just a note in the README to explain what it is!